zziplib/PSPBUILD

```diff
@@ -9,25 +9,22 @@ license=('ZLIB')
 depends=('zlib')
 makedepends=()
 optdepends=()
-source=("git+https://github.com/pspdev/psp-ports.git#commit=5ee159bccf6b9539d69b5f43bba0eff0c75e6f11")
+source=("git+https://github.com/gdraheim/zziplib.git#commit=a82f98f94a2cc365fad71eee02d7dd50c0810844")
```

Comment:
Please use the upstream tar.gz if possible

Reply:
I can't because the needed changes to make PSP to compile is there, I have done a PR for it, and they merged already.
So until they do a new version we can't use the `tar.gz`

Reply:
Makes sense, thanks